### DISCUSSION TITLE
Generate public from secret key

### DISCUSSION THREAD

**Discussion by tobiasjungnickel** [krzyzanowskim/ObjectivePGP#199] [General]
- ObjectivePGP version: 0.14.0
- Platform (iOS/macOS): iOS 12.3.1
- Installation method (CocoaPods/Carthage/...): CocoaPods 

# Description

Is there a way to generate the public `PartialKey` from a `PGPKey`containing the secret `PartialKey` only?

**Comment by krzyzanowskim** [comment]
no. keypair is generated at once

**Comment by tobiasjungnickel** [comment]
ok! Can you please mark this ticket as a feature request.
